src/StreamingDataFrames/streamingdataframes/models/serializers/base.py

```diff
@@ -65,7 +65,7 @@ def _to_dict(self, value: Any) -> Union[Any, dict]:
 
     @abc.abstractmethod
     def __call__(self, *args, **kwargs) -> Any:
-        ...
+        pass
```

Comment:
Is there a reason why we can't use `...` instead of `pass`?

Reply:
In most cases you still can and I was wrong here (IDE was not very clear about where the error was); I added them back where applicable. You can't do it with respect to types, so those remain replaced =)